.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "java.compile.nullAnalysis.mode": "automatic"
```

Comment:
Did you intend to commit this file? It looks unrelated to the PR topic

Reply:
did not, let me fix.